tests/python/test_impactx.py

```diff
@@ -26,6 +26,11 @@ def test_impactx_fodo_file():
     sim.init_lattice_elements_from_inputs()
 
     sim.evolve()
+    # test calculation of reduced beam characteristics
+    reduced_beam_data = sim.reduced_beam_characteristics()
```

Comment:
@ax3l, do we want a separate test or is it fine to include this here?

Reply:
Here is good, please add a validation for the final beam here as done in these lines:
https://github.com/ECP-WarpX/impactx/blob/c29f1ec3f6261963c45b1063afac8d496ebd9e7f/examples/fodo/analysis_fodo.py#L88-L104

Reply:
Thanks! Final beam validation was added